rfcs/033-lineage-graph.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: lineage-graph-visualization
+- Start Date: 2021-04-01
+- RFC PR: [amundsen-io/rfcs#33](https://github.com/amundsen-io/rfcs/pull/33)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Amundsen Lineage Stage 1 - Graph Visualization
+
+## Summary
+
+This is the next in the series of https://github.com/amundsen-io/rfcs/pull/25
+
+Implement a navigable canvas-based graph visualization for lineage. Lineage is traditionally represented this way, suboptimally, in the various existing tools like CompilerWorks, Castor, Apache Atlas, etc.
+
+This will replace the existing `Upstream` and `Downstream` tabs on the table detail page, and introduce a new CTA "Lineage" on the table detail page.
```

Comment:
Given the lineage tab has just been merged and checked in, does it make sense to remove and replace with this without product verification ? 

cc @danwom

Reply:
another way to ask is that whether it makes sense to have both until we found the graph UI is ready to replace the tab view .

Reply:
The idea is to keep the list-based view in place for the time being, and the end state of this feature is to actually replace the list-based view with this graph visualization. 
I updated the summary section to make it explicit, and also added the configuration changed in the "Technical Details" section on how I think it will be during the development of this feature.